app/routes/projects/index.tsx

```diff
@@ -38,6 +52,9 @@ export const loader: LoaderFunction = async ({ request }) => {
   const missing = url.searchParams.getAll("missing");
   const field = url.searchParams.get("field") || "";
   const order = url.searchParams.get("order") || "";
+  const ongoingStatuses = (await getProjectStatuses()).filter((status) => status.stage === ongoingStage);
+  const ideaStatuses = (await getProjectStatuses()).filter((status) => status.stage === ideaStage);
+
```

Comment:
What if wwe only do the DB search only once and then filter the results? i.e.
```
  const statuses = await getProjectStatuses()
  const ongoingStatuses = statuses.filter((status) => status.stage === ongoingStage);
  const ideaStatuses = statuses.filter((status) => status.stage === ideaStage);
```
That way we only access the DB once and it could be faster at the end

Reply:
Yes, that is inadequate, thanks for caching it!